sys/net/crosslayer/ng_netif/ng_netif_addr_parse.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+
+#include <string.h>
+
+#include "net/ng_netif.h"
+
+static uint16_t _parse_byte(char *str)
```

Comment:
I'm not sure… it's an internal function and pretty straightforward IMHO... Except you don't agree with me on that part